common/models/access-token.js

```diff
@@ -78,6 +78,77 @@ module.exports = function(AccessToken) {
     });
   });
 
+  AccessToken.getAccessTokenId = function(req, options) {
```

Comment:
Since the method moved its impossible to do a diff...? Is this unchanged otherwise?